src/Frontend/Core/Engine/TemplateModifiers.php

```diff
@@ -11,6 +11,7 @@
 
 use Frontend\Core\Engine\Model as FrontendModel;
 use Frontend\Core\Engine\Block\Widget as FrontendBlockWidget;
+use Frontend\Core\Language\Language;
```

Comment:
I think that's only an issue if you explicitly use that class. If it's implicitly used (like in this case) it's no problem. I'm using this code in production right now :)